_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -993,14 +1040,14 @@ IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low valu
 
 Set the variables for the `account-baseline-root` module in this environment in the `inputs = { ... }` block of `terragrunt.hcl`:
 
-.infrastructure-live/root/_global/account-baseline/terragrunt.hcl
+.infrastructure-live/prod/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
 locals {
-  aws_region = "us-east-1"
+  aws_region = local.aws_region
 
   accounts = {
-    root = "216044045972"
+    root = local.accounts.root
```

Comment:
this is a recursion 😂  it is referencing itself.

Reply:
hahahha 😬 let's change it to <ROOT_ACCOUNT_ID>

Reply:
Ok, the previous IDs were random ones

Reply:
ahh! I didn't know that - they looked real. I'll actually replace it with `0123456789` to be on the safe side